src/URI.js

```diff
@@ -780,6 +780,21 @@
       throw new TypeError('URI.addQuery() accepts an object, string as the name parameter');
     }
   };
+
+  URI.setQuery = function(data, name, value) {
```

Comment:
do you mind adding tests for this to test/test.js?

Reply:
Sure, sorry so the delay, I got really busy there days.

Reply:
no worries… I'm leaving for 2 week vacation in 1h, sorry for *that* delay :(

Reply:
Tests added, happy vacation!